### PR TITLE
🐛 Frontend: fix Shared with sorting

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

- [x] Bugfix: do not override collaborator information
- [x] Refactor sorting by accessRights methods
- [x] Shorter NavBar: 60px -> 50px

## Related issue/s

fixes https://github.com/ITISFoundation/osparc-issues/issues/856

### CODE REVIEW

**Review by mrnicegyu11 — APPROVED** [review]
gracias

File: services/static-webserver/client/source/class/osparc/component/permissions/Permissions.js [hunk 1]
```diff
@@ -38,6 +38,41 @@ qx.Class.define("osparc.component.permissions.Permissions", {
...
+      return 0;
+    },
+
+    sortStudyOrServiceCollabs: function(a, b) {
+      const aAccessRights = a["accessRights"];
+      const bAccessRights = b["accessRights"];
+      let sorted = null;
+      if ("delete" in aAccessRights) {
+        // studies
+        sorted = this.self().sortByAccessRights(a, b);
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/component/permissions/Permissions.js:63]: Q: so ``sorted`` is an integer  that resolves whether ``a<=b`` (==1:True, <=0:False)??

**Review by pcrespov — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/component/permissions/Permissions.js (hunk 1 above)
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/component/permissions/Permissions.js:63]: -1, 1: sorting
0: no sorting

File: services/static-webserver/client/source/class/osparc/component/permissions/Permissions.js [hunk 2]
```diff
@@ -38,6 +38,41 @@ qx.Class.define("osparc.component.permissions.Permissions", {
     this.__getCollaborators();
   },
 
+  statics: {
+    sortByAccessRights: function(a, b) {
+      const aAccessRights = a["accessRights"];
+      const bAccessRights = b["accessRights"];
+      if (aAccessRights["delete"] !== bAccessRights["delete"]) {
+        return bAccessRights["delete"] - aAccessRights["delete"];
```
[the remark this answers is not in the document]
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/component/permissions/Permissions.js:46]: an integer: -1, 1 or 0

**Pull request merged**